src/linear_solver_routines/block_col_projection.jl

```diff
@@ -0,0 +1,64 @@
+# This file is part of RLinearAlgebra.jl
+"""
+    LinSysBlkColGent <: LinSysBlkColProjection
+
+A mutable structure that represents a standard block column projection method that projections 
+a previous iterate onto the columnspace of an inputted block. This method can be viewed as 
+a generalization to block coordinate descent. The updates to this are computed
+using Gentleman's Algorithm.
+
+For more information on Gentleman's see:
+Miller, Alan J. “Algorithm AS 274: Least Squares Routines to Supplement Those of Gentleman.” 
+Journal of the Royal Statistical Society. Series C (Applied Statistics), 
+vol. 41, no. 2, 1992, pp. 458–78. JSTOR, https://doi.org/10.2307/2347583. Accessed 8 July 2024.
```

Comment:
What citation format are you using? Why do you need an access date?

Reply:
I was using MLA, and just grabbing the citation from ResearchGate or one of those types of websites.